examples/pytorch/CaptumExample/conda.yaml

```diff
@@ -0,0 +1,18 @@
+channels:
+- pytorch
+- defaults
+- conda-forge
+dependencies:
+- python=3.8.2
+- pytorch
+- torchtext
+- pip
+- pip:
+  - mlflow
+  - pandas
+  - captum
+  - sklearn
+  - prettytable
+  - spacy
```

Comment:
Are we using spacy?

Reply:
@harupy  Yes, we have used spacy in this example.

Reply:
Looks like we no longer use `spacy`

Reply:
@harupy Yeah, now we are not using spacy. I  will update this. Thanks.

Reply:
@harupy yes you are right.